upup/models/cloudup/resources/addons/node-authorizer.addons.k8s.io/k8s-1.10.yaml.template

```diff
@@ -147,8 +147,8 @@ spec:
       volumes:
         - name: config
           hostPath:
-            path: /srv/kubernetes/node-authorizer
-            type: DirectoryOrCreate
+            path: /etc/srv/kubernetes/node-authorizer/
+            type: Directory # Don't create, as can't guarantee permissions
```

Comment:
So this reason behind why this is needed as `DirectoryOrCreate` is the cluster validation code .. As you roll out the node authorizer (daemonset), which is kicked off by `channels` in protokube which can happen long before the nodes have been rolled. What will happen is the daemonset get pushed out, but the host directory doesn't exist yet, causing the pods to `Error`. As the roll out finishes on the first node it calls cluster validation, which will now fail due to pods within the `kube-system` namespace .. It will essentially block any roll out.